vrk-kpa/suomidigi
# Suopa 695 shortcuts

To test: 
- `make fresh`
- Go to https://suomidigi.docker.sh
- Check that the shortcuts have been moved from sidebar to footer
- It should be something like this: 
![vrk-suomidigi-oct-2019-uusi 2019-11-08 11-21-42](https://user-images.githubusercontent.com/1712902/68464982-3e8c2a80-021a-11ea-8c0b-b68cee3fa1e3.jpg)
- Check that the shortcuts have been removed from mobile menu
